homeassistant/components/vicare/water_heater.py

```diff
@@ -152,3 +171,33 @@ def current_operation(self) -> str | None:
         if self._current_mode is None:
             return None
         return VICARE_TO_HA_HVAC_DHW.get(self._current_mode, None)
+
+    def set_dhw_circulation_pump_schedule(self, schedule) -> None:
+        """Service function to set schedule for dhw circulation pump directly."""
+
+        try:
+            schedule_json = json_loads_object(schedule)
+        except Exception as error:
```

Comment:
don't catch raw exceptions, we know what we can expect from such call, so let's catch them